core/App/ehsm_provider.cpp

```diff
@@ -962,6 +981,18 @@ ehsm_status_t VerifyQuote(ehsm_data_t *quote,
 
     sgx_ql_qe_report_info_t qve_report_info;
     uint8_t nonce[16] = {0};
+    uint8_t mrsigner_quote[SGX_HASH_SIZE] = {0};
+    uint8_t mrenclave_quote[SGX_HASH_SIZE] = {0};
+    uint8_t mrsigner_base_arry[SGX_HASH_SIZE] = {0};
+    uint8_t mrenclave_base_arry[SGX_HASH_SIZE] = {0};
```

Comment:
why define so many arrays for them?

Reply:
I'll remove them and  realize it in another way.